Comments/components/commentBrowser.js

```diff
@@ -25,12 +18,22 @@ export class CommentBrowser extends Window {
   }
 
   static isOpen () {
-    return instance;
+    return instance && $world.get('comment browser');
+  }
+
+  static async removeCommentForMorph (updatedComment, morph) {
+    await instance.removeCommentForMorph(updatedComment, morph);
```

Comment:
```suggestion
  static async removeCommentForMorph (commentToRemove, morph) {
    await instance.removeCommentForMorph(commentToRemove, morph);
```
Warum heißt es updatedComment, wenn der Kommentar gelöscht werden soll?